packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -104,6 +114,8 @@ export const ChatMessageComponentAsMessageBubble = (props: ChatMessageComponentA
           onEditClick={onEditClick}
           onRemoveClick={onRemoveClick}
           strings={strings}
+          messageThreadReadCount={message.readNumber ? message.readNumber : 0}
+          participantCountNotIncludingSelf={participantCountNotIncludingSelf ? participantCountNotIncludingSelf : 0}
```

Comment:
Why default to 0 here? Since `participantCountNotIncludingSelf` is a prop being forwarded, it would be better to default higher up where this is first defined, or lower in the component that uses it.

Reply:
participantCountNotIncludingSelf is an optional prop so I need to give it a default value here, what's a better way for giving default values to optional prop?

Reply:
can we not say its optional and self the value to 0 if we dont pass in anything? it would clean up this area of the code.
https://www.javatpoint.com/typescript-function-parameter#:~:text=TypeScript%20Function%20Parameter%201%20Optional%20Parameter.%20JavaScript%20allows,zero%20or%20more%20values%20to%20a%20function.%20
![image](https://user-images.githubusercontent.com/79329532/154314627-3686c49f-1981-4b0b-8114-c4eef143d028.png)

Reply:
updated :)